certified-connectors/dvelop/README.md

```diff
@@ -35,4 +35,7 @@ As part of this sample following triggers are supported:
 ## Supported Actions
 As part of this sample following actions are supported:
 * `Store document`: Stores a document in d.velop documents
+* `Update document`: Updates a document or the properties of a document in d.velop documents
+* `Create dossier`: Creates a dossier in d.velop documents
```

Comment:
@BeritHorst do you mean `create a folder` in this one? If yes, can you please change it to `folder`?
Please review the readme.md file, everything should be according to [the readme.md template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md).